Project-Service-implementation-Script/common_config/config.ini

```diff
@@ -0,0 +1,133 @@
+; keyclockapibody = '{"email": "rahul@yahoo.com","password": "password"}'
+; email = rahul@yahoo.com
+; password = password
+; internal-access-token = 8c3a94f0931e01a4940a
+; keyclockapiurl = user/v1/account/login
+
+
+; DEV_testing
+host = https://dev.elevate-apis.shikshalokam.org/
+host1 = https://dev.elevate-apis.shikshalokam.org/
+internal_kong_ip = https://dev.elevate-apis.shikshalokam.org/
+keyclockapibody = '{"email": "pavan@abc.com","password": "Pavan@Pavan001"}'
+email = pavan@abc.com
+password = Pavan@Pavan001
+internal-access-token = Fqn0m0HQ0gXydRtBCg5l
+keyclockapiurl = elevate-interface/v2/account/login
+
+authorization = Bearer  # ADD_YOUR_TOKEN_HERE
+authorizationforhost = Bearer # ADD_YOUR_TOKEN_HERE
+appname = diksha
+x-channel-id = 0125747659358699520
+keyclockapicontent-type = application/x-www-form-urlencoded
+content-type = application/json
+userinfoapiurl = api/user/v1/account/search
+searchforlocation = project/v1/programs/create
+criteriauploadapiurl = private/mlsurvey/api/v1/criteria/upload
+themeuploadapiurl = private/mlsurvey/api/v1/frameworks/uploadThemes/
+solutioncreationapiurl = private/mlsurvey/api/v1/observations/importFromFramework
+surveysolutioncreationapiurl = private/mlsurvey/api/v1/surveys/createSolutionTemplate
+questionuploadapiurl = private/mlsurvey/api/v1/questions/bulkCreate
+criteriarubricuploadapiurl = private/mlsurvey/api/v1/solutions/uploadCriteriaRubricExpressions/
+themerubricuploadapiurl = private/mlsurvey/api/v1/solutions/uploadThemesRubricExpressions/
+importsurveysolutiontemplateurl = private/mlsurvey/api/v1/surveys/importSurveryTemplateToSolution/
+importsurveysolutiontoprogramurl = private/mlsurvey/api/v1/surveys/mapSurverySolutionToProgram/
+solutiontoprogrammappingapiurl = private/mlsurvey/api/v1/solutions/importFromSolution
+fetchprograminfoapiurl = project/v1/programs/list?page=1&limit=5&search=
+fetchsolutiondetails = project/v1/solutions/list?page=1&limit=100&search=&type=observation&subType
+frameworkcreationapi = private/mlsurvey/api/v1/frameworks/create
+solutionupdateapi = project/v1/solutions/update/
+listofrolesapi = private/mlsurvey/api/v1/userRoles/list
+ferchsolutioncriteria = private/mlsurvey/api/v1/solutionDetails/criteria/
+; Create
+projectuploadapi = project/v1/project/templates/bulkCreate
+taskuploadapi = project/v1/project/templateTasks/bulkCreate/
+projectsolutioncreationapi = project/v1/solutions/create
+mapsolutiontoproject = project/v1/project/templates/importProjectTemplate/
+fetchsolutiondoc = project/v1/solutions/getDetails/
+programcreationurl = project/v1/programs/create
+pdpmurl = private/mlsurvey/api/v1/userExtension/bulkUpload
+fetchlink = project/v1/solutions/fetchLink/
+courseprogrammapping = private/mlcore/api/v1/solutions/create
+readcourseurl = api/content/v1/read/
+fetchsolutiondump = private/mlcore/api/v1/solutions/getDetails/
+fetchorgdetails = api/org/v1/search
+fetchprojectlist = private/mlprojects/api/v1/library/categories/projects
```

Comment:
@MallanagoudaB why we are still having mlprojects end point?

Reply:
please refer config.sample